install.sh

```diff
@@ -60,11 +59,20 @@ fi
 if [ ! -z ${VIRTUAL_ENV+x} ]; then
     echo "Detected virtual environment $VIRTUAL_ENV"
     pip install -e .
+elif [ ! -z ${TYKKY_PREFIX+x} ]; then
+    echo "======================="
+    echo "Using TYKKY environment"
+    echo "======================="
+    echo "WARNING: The use of a tykky environment for using ESM-Tools is currently not supported. Use at your own risk"
```

Comment:
Well...we kinda do support it? Since we wrote this section of the install script and a handbook section?